epoch2d/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Did you mean for this file to be added?

Reply:
No I didn't - I saw it, and thought I took it off the commit.